cc.arduino.IDE2.json

```diff
@@ -138,8 +138,8 @@
                     "only-arches": [
                         "x86_64"
                     ],
-                    "url": "https://github.com/arduino/arduino-ide/releases/download/2.0.0/arduino-ide_2.0.0_Linux_64bit.zip",
-                    "sha256": "85f889f4447987edd0e2402175e8d46bbfd6e199a12063819dc8e993bb1b7bf3",
+                    "url": "https://github.com/arduino/arduino-ide/releases/download/2.0.1/arduino-ide_2.0.1_Linux_64bit.zip",
```

Comment:
Not ready for publish. Has bugs in flatpak environement

Reply:
These bugs are suspected to be upstream (by me) awaiting response from upstream.
(https://github.com/arduino/arduino-ide/issues/1586)

Reply:
Merging as these are upstream